### ISSUE TITLE
Composer install

### ISSUE THREAD

**Issue by mikeymike** [php-school/php-workshop#156]
@AydinHassan wanted to get your thoughts on this 😄 

Currently we run a composer install on the solution files but we only do this if the composer.lock file is present. 

Currently learnyouphp Dependency Heaven exercise is broken because of a library update where they changed the casing in the output. 

The quick solution would be to update the dependencies in leanyouphp but this may become a wider issue with minor version increments. Instead I'm thinking we drop the composer.lock file from solutions, much like the concept behind doing this with libraries so that we're not forcing a single version but install the latest available at the time (meeting requirements ofc).

So instead of checking for the existance of the lock file before running the composer install command we just check for the json file. Then in learnyouphp we can just drop the lock file and problem solved.

**Comment by AydinHassan** [comment]
@mikeymike sounds pretty reasonable to me, only issue would be that they might accidentally introduce BC breaks tho? Or on purpose for that matter, then our solution might not work anymore?

**Comment by mikeymike** [comment]
Yeah that would be a concern. It's a trade-off either way I think. In the current case a libs non BC update caused it to break, so it's a trade-off to whether a non BC breaks could break a workshop more often/common than an accidental BC introduction 🤔
